pkg/storage/storage.go

```diff
@@ -426,6 +427,11 @@ func (s *storageModule) ensureCache() error {
 		return syscall.Mount("", "/var/cache", "tmpfs", 0, "size=500M")
 	}
 
+	log.Info().Msgf("set cache quota to %d GiB", cacheSize/gib)
+	if err := cacheFs.Limit(cacheSize); err != nil {
```

Comment:
Afaik this call is already done by the call to createSubvol?

Reply:
Yes, but since all the running node already have the cache created. I added this line here to make sure we always have the right quota set on the cache subvolume in all cases.

Reply:
alrighty